src/onConsentChange.test.js

```diff
@@ -185,4 +185,33 @@ describe('under TCFv2', () => {
 			expect(callback).toHaveBeenCalledTimes(2);
 		});
 	});
+
+	it('invokes callbacks only if there is a user action', async () => {
```

Comment:
My understanding is that `onConsentChange` will still be called in the case we have stored a previous valid consent choice, so not _only_ if there is a user action.

Should this be something like `'if cmpuishown callbacks are invoked only after user action'`?